homeassistant/components/hive/__init__.py

```diff
@@ -1,17 +1,32 @@
-"""Support for the Hive devices."""
+"""Support for the Hive devices and services."""
+from functools import wraps
 import logging
 
 from pyhiveapi import Pyhiveapi
 import voluptuous as vol
 
-from homeassistant.const import CONF_PASSWORD, CONF_SCAN_INTERVAL, CONF_USERNAME
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    ATTR_TEMPERATURE,
+    CONF_PASSWORD,
+    CONF_SCAN_INTERVAL,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.discovery import load_platform
+from homeassistant.helpers.dispatcher import async_dispatcher_connect, dispatcher_send
+from homeassistant.helpers.entity import Entity
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = "hive"
 DATA_HIVE = "data_hive"
+SERVICES = ["Heating", "HotWater"]
+SERVICE_BOOST_HOTWATER = "boost_hotwater"
```

Comment:
Noticed during the review of the documentation: `hotwater` is not one word. So I expected it to be `hot_water` ?